docs/Zigbee.md

```diff
@@ -452,18 +453,15 @@ Command `ZbLight` configures a Zigbee device to be Alexa controllable. Specify t
 * `4` RGBW Light
 * `5` RGBCW Light, RGB and  Cold/Warm White
 
-To set the light, use `ZbLight <device>,<nb_of_channels`.
-Ex:
+To set the light, use `ZbLight <device>,<nb_of_channels>`. Example:
 ```haskell
 ZbLight 0x1234,2
 ZbLight Kitchen_Light,1   (see ZbName)
 ```
 
 Once a light is declared, Z2T will monitor any change made to the light via Z2T or via remotes, either from a direct message or via a group message. Z2T will then send a read command to the light, between 200ms and 1000ms later, and memorize the last value.
 
-To read the last known status of a light, use `ZbLight <device>`
-
-!!! example
+To read the last known status of a light, use `ZbLight <device>`. Example:
```

Comment:
restore formatting

Reply:
Restored original formatting.
Might be beneficial to align all the formattings, but I cannot find out any how-to for the !!! markdown usage...